src/cryptoadvance/specter/util/seedqr.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import re
+from embit import bip39
+from embit.wordlists.bip39 import WORDLIST
+from typing import List
+
+logger = logging.getLogger(__name__)
+
+
+"""
+    SeedQR specification:
+    https://github.com/SeedSigner/seedsigner/blob/dev/docs/seed_qr/README.md
+"""
+
+
+def parse_standard_seedqr(
+    digitstream: str, wordlist: List[str] = WORDLIST
+) -> List[str]:
+    """
+    Reads a Standard SeedQR digitstream and returns its associated mnemonic
+    """
+    # The digitstream should be either:
+    #   * 48 digits (12-word mnemonic)
+    #   * 96 digits (24-word mnemonic)
+    if not re.search("^[0-9]{48}$|^[0-9]{96}$", digitstream):
+        logger.warn(f"Invalid Standard SeedQR datastream: {digitstream}")
+        return None
+
+    # Parse the digitstream, 4 digits at a time
+    mnemonic = []
+    for i in range(0, int(len(digitstream) / 4)):
+        wordlist_index = int(digitstream[i * 4 : i * 4 + 4])
+        mnemonic.append(WORDLIST[wordlist_index])
+
+    return mnemonic
```

Comment:
Perhaps, return the result of `" ".join(mnemonic)` here to have it consistent with `parse_compact_seedqr`? That would also make the handling at `wallets_api.py` the same again.